lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -547,6 +548,29 @@ public void flushIndexing(final ObjectTypeParsingRules rule) throws IOException
         return ImmutableMap.copyOf(ret);
     }
 
+    public boolean lookupParentId(final String type, final GUID guid) throws IOException {
+        final String indexName = indexNamePrefix + type + "*";
+        final Set<GUID> guids = new HashSet<>();
+        guids.add(guid);
+        Map<String, Object> doc =
```

Comment:
Could you add a comment of the actual query here?
e.g.   like for checkParentGuidsExist    // doc = {"query": {"bool": {"filter": [{"term": {"prefix": prefix}}]}}}